packages/lwt-pipeline/lwt-pipeline.0.1/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+synopsis: "Pipeline library for Lwt"
+description: """
+A library to handle pipelines (series of transformation) with support for
+Lwt-based concurrency.
+"""
+maintainer: "Raphaël Proust <code@bnwr.net>"
+authors: "Raphaël Proust <code@bnwr.net>"
+license: "MIT"
+homepage: "https://gitlab.com/nomadic-labs/lwt_pipeline"
+bug-reports: "https://gitlab.com/nomadic-labs/lwt_pipeline/issues"
+dev-repo: "git+https://gitlab.com/nomadic-labs/lwt_pipeline.git"
+depends: [
+	"ocaml" { >= "4.03" }
+	"ocamlfind" { build }
+	"dune" { build & >= "1.7" }
+	"lwt"
+	]
+build: ["dune" "build" "-p" name]
```

Comment:
```suggestion
build: ["dune" "build" "-p" name "-j" jobs]
```
It would be perfect to let users configure how much cores they want to dedicate in building this package

Reply:
I pushed a new version that implements both of your suggestions.